README.md

```diff
@@ -50,19 +50,19 @@ DESCRIPTION
 
 ## Step-by-Step guide
 
-Examples are provided for driver version 410.78. All commands are runned as root.
+Examples are provided for driver version 418.43. All commands are runned as root.
 
 ### Download driver
 
-[https://download.nvidia.com/XFree86/Linux-x86\_64/410.78/NVIDIA-Linux-x86\_64-410.78.run](https://download.nvidia.com/XFree86/Linux-x86_64/410.78/NVIDIA-Linux-x86_64-410.78.run)
+[https://download.nvidia.com/XFree86/Linux-x86_64/418.43/NVIDIA-Linux-x86_64-418.43.run](https://download.nvidia.com/XFree86/Linux-x86_64/418.43/NVIDIA-Linux-x86_64-418.43.run)
```

Comment:
Underscores in text should be escaped with a backslash, otherwise some Markdown (but not GitHub one) may interpret text between underscores as emphasis
```suggestion
[https://download.nvidia.com/XFree86/Linux-x86\_64/418.43/NVIDIA-Linux-x86\_64-418.43.run](https://download.nvidia.com/XFree86/Linux-x86_64/418.43/NVIDIA-Linux-x86_64-418.43.run)
```

Reply:
Screenshot with example of issue I'm speaking about:

![Screenshot](https://gist.githubusercontent.com/Snawoot/2d2e3c3257c8a7f2feba74d83500c43a/raw/da0704f781ba28164b596a04bdfcdc99a1104aef/2019-03-10-200219_1039x558_scrot.png)